app/helpers/lessons_helper.rb

```diff
@@ -3,9 +3,9 @@ def github_edit_url(lesson)
     github_link("curriculum/edit/main#{lesson.github_path}")
   end
 
-  def github_report_url(lesson)
+  def github_report_url(lesson, path)
     # rubocop:disable Layout/LineLength
-    github_link("curriculum/issues/new?assignees=nil&labels=Status%3A+Needs+Triage&projects=&template=suggestion.yaml&title=#{lesson.title}%3A+%3CShort+description+of+your+suggestion%3E")
+    github_link("curriculum/issues/new?assignees=nil&labels=Status%3A+Needs+Triage&projects=&template=suggestion.yaml&title=#{lesson.title}%3A+%3CShort+description+of+your+suggestion%3E&lesson-link=https://www.theodinproject.com#{path}")
```

Comment:
Unrelated to your work, but these query params are difficult to read and could use a refactor. Would you mind changing the method body to this please Asartea ?

```suggestion
params = {
      labels: 'Status: Needs Triage',
      template: 'suggestion.yaml',
      title: "#{lesson.title}: <Short description of your suggestion>",
      lesson: lesson_url(lesson)
    }

    github_link("curriculum/issues/new?#{params.to_query}")
```

I'm not sure if we need the assignees or projects params. They had blank values so we might get away without them 🤞